NSS-Day-Cohort-42/rare-API-media-moguls
# Updated bash script file to repopulate database after nuking.

#### Changes Made
1. Modified file `nuke.sh`.
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-bash
```
2. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
